### ISSUE TITLE
Wrong request uri?

### ISSUE THREAD

**Issue by d0m3n1cc** [kserve/kserve#3497]
I used to have kserve sdk version 0.10.0, but I had to update it to version 0.11.2 due to a kubeflow update in a cluster. I used to get the correct url using self.client.get(name, namespace=namespace)["status"]["address"]["url"], as it used to give me the exact url for request ({url} + /v1/models/{name}:predict), but now I am only getting the {url} part, without the complement to the true request endpoint.
Am I doing something wrong here? I have to manually add the "/v1/models/{name}:predict" part in the url as I can't find this information available anywhere.

I tested the 0.12.0 version and I am getting the same response.

**Comment by sivanantha321** [comment]
Yes, the url is changed in 0.11 to make it standard across both serverless/raw installation. https://github.com/kserve/kserve/issues/2874

**Comment by d0m3n1cc** [comment]
@sivanantha321  I'm sorry, there's something missing for me. As I am understanding what you said, I should be able to set the uri using "kserve_client.create()" am I right? But I cannot find in the method description any place to define this. 
Considering that the default value is "{url}/v1/models/{name}:predict" as it still happens, is there any method to get this information via sdk?

**Comment by sivanantha321** [comment]
@d0m3n1cc No, You can't set url using `kserve_client.create()`. The rest of the url is removed because, the user may want to control over which endpoint to call. What is your use case ?

**Comment by d0m3n1cc** [comment]
@sivanantha321 I am using kserve sdk to create inference services and I just want to get the real url to make the inference call, but the url that I am receiving as response to 'get inference' is not giving me the true url, as opposed to the old version of the sdk. Is there a way I can get it? You said "the user may want to control", but how am I supposed to control this?
